### PR TITLE
households and TPI

### PR DESCRIPTION
**Pull request by LeiliPR**
I think I am done with TPI, I made changes on household but I am not sure about that labor. So I will let the ones that wrote that code, change it or keep it.

### CODE REVIEW

File: scripts/TPI.py
Comment (jtregde) [inline scripts/TPI.py:4]: Should we have this while loop inside a function, like `solve_TP` or something so we can call it from the execute file?

Comment (LeiliPR) [reply scripts/TPI.py:4]: yes, go ahead and make the change please.

File: scripts/households.py
Comment (saharnazb) [inline scripts/households.py:8]: @LeiliPR I think it's better to rename this to FOCs to be consistent with other scripts where `households.py` is used (e.g. SS)

File: scripts/TPI.py
Comment (saharnazb) [inline scripts/TPI.py:4]: @LeiliPR Define `TPI` as a function so that we can call this in `execute.py`. Just the same as what we did in class

File: scripts/TPI.py
Comment (saharnazb) [inline scripts/TPI.py:38]: @LeiliPR The same here:
hh.FOCs

File: scripts/TPI.py
Comment (saharnazb) [inline scripts/TPI.py:3]: @LeiliPR As you are using functions from `households.py` , ... import them
e.g. 
import households as hh
import firm

**Review by saharnazb — CHANGES_REQUESTED** [review]

**Comment by jdebacker** [comment]
@LeiliPR My suggestion here is to listen to your colleagues suggestions on `TPI.py` and make those quick changes.

I would recommend that you just keep the `households.py` untouched in this PR.  It looks like you might be deleting some important stuff there.  You can just copy and paste the code for `households.py` over from the master branch to this branch and then commit that change to this branch.

File: scripts/TPI.py
Comment (saharnazb) [reply scripts/TPI.py:4]: @LeiliPR I would recommend you do it because we do not have access to the code before approval of pull request unless copy and paste your code in a new file. I think you can do it quicker

**Comment by LeiliPR** [comment]
Sure, give me one minute, I am trying to apply all changes.

File: scripts/TPI.py
Comment (LeiliPR) [reply scripts/TPI.py:4]: @saharnazb Sure, will do all ASAP

**Comment by LeiliPR** [comment]
As @miltonash (Milton) is working on this script to fix the problems, I am waiting for him to make changes, then I will close this pull request and he will open a new Pull request with fixed TPI.

**Pull request closed without merging**
